Sources/Workspace/InitPackage.swift

```diff
@@ -190,6 +233,12 @@ public final class InitPackage {
                         .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.2.0"),
                     ]
                 """)
+            } else if packageType == .macro {
+                pkgParams.append("""
+                    dependencies: [
+                        .package(url: "https://github.com/apple/swift-syntax.git", branch: "main"),
```

Comment:
We should make sure to have a follow-up task to change this from `main` once we have a concrete version that people should be using.

Reply:
Right. We should get that once we branch for Swift 5.9.